pkg/interface/src/views/landscape/components/Graph/GraphContent.tsx

```diff
@@ -283,7 +283,7 @@ const renderers = {
   },
   paragraph: ({ children }) => {
     return (
-      <Box fontSize={1} lineHeight="tall">
+      <Box color="black" fontSize={1} lineHeight="tall">
```

Comment:
Someone has to stop Liam from using Box like this. Can you wrap this in a `<Text>` child?

Reply:
He Simply Must Be Stopped